docs/manuals/auth/authentication.md

```diff
@@ -222,19 +205,52 @@ authenticator = Authenticator("taipy", passwords=passwords)
 Note that these values are the one resulting from the example of  the
 [creating hashed passwords](#creating-hashed-passwords) section below.
 
-Calling `login("user1", "pass123")^` will result in a valid `Credentials^` 
+Calling `login("user1", "pass123")^` will result in a valid `Credentials^`
 instance where the assigned roles is an empty set.
 
-Of course you can combine both roles and password for any given user, using
+!!! Note "Using Taipy configuration to set default authenticator"
+    To set the *default authenticator* to a Taipy authenticator with passwords, you can use the authentication
+    configuration either in Python or Toml:
+
+    === "Python configuration"
+
+        ```
+        Config.configure_authentication(protocol="taipy",
+                                        passwords={
+                                            "user1": "eSwebyvpEElWbZNTNqpW7rNQPDPyJSm",
+                                            "user2": "JQlZ4IXorPcJYvMLFWE/Gu52XNfavMe"})
+        taipy.auth.login("user1", "pass123")  # returns a valid Credentials instance
+        taipy.auth.login("user1", "anything_else")  # raise an InvalidCredentials exception
+        ```
+
+    === "TOML configuration"
+
+        ```toml title="config.toml"
+
+        [AUTHENTICATION]
+        protocol="taipy"
+
+        [AUTHENTICATION.passwords]
+        user1 = "eSwebyvpEElWbZNTNqpW7rNQPDPyJSm"
+        user2 = "JQlZ4IXorPcJYvMLFWE/Gu52XNfavMe"
+        ```
+
+        ```python title="main.py"
+        Config.load("config.toml")
+        taipy.auth.login("user1", "pass123")  # returns a valid Credentials instance
+        taipy.auth.login("user1", "anything_else")  # raise an InvalidCredentials exception
+        ```
+
+Of course, you can combine both roles and password for any given user, using
 both the *roles* and *passwords* arguments of the
 [`Authenticator` constructor](Authenticator.__init__()^), or using its
 *config* argument:
 ```
 users={
     "roles": {
-      "user1": "role1",
-      "user2": "role2",
-      "user3": ["role1", "role2"]
+        "user1": ["role1", "TAIPY_READER"],
```

Comment:
I really wanted to show that a user can have a single role, and then the array construct is not mandatory.

Reply:
Yes, it makes sense. Let me change this.